README.md

```diff
@@ -1,3 +1,280 @@
+styleLoaders:
+  - style
+  - css
+  - sass
+
+# You can apply loader params here:
+  - sass?outputStyle=expanded
+```
+
+#### `extractStyles`
+
+Default: `false`
+
+Extract styles to stand-alone css file using `extract-text-webpack-plugin`.
+
+```yaml
+extractStyles: false
+
+# Different settings for different environments can be used,
+# It depends on value of NODE_ENV environment variable
+env:
+  development:
+    extractStyles: false
+  production:
+    extractStyles: true
+```
+
+This param can also be set to `true` in webpack config:
+
+```js
+entry: [ 'bootstrap-loader/extractStyles', './app' ]
+```
+
+#### `preBootstrapCustomizations`
+
+Default: `disabled`
+
+Customize Bootstrap variables that get imported before the original Bootstrap variables. Thus, derived Bootstrap variables can depend on values from here. See the Bootstrap [`_variables.scss`](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss) file for examples of derived Bootstrap variables.
+
+```yaml
+preBootstrapCustomizations: ./path/to/bootstrap/pre-customizations.scss
+```
+
+#### `bootstrapCustomizations`
+
+Default: `disabled`
+
+This gets loaded after bootstrap variables is loaded. Thus, you may customize Bootstrap variables based on the values established in the Bootstrap [`_variables.scss`](https://github.com/twbs/bootstrap-sass/blob/master/assets/stylesheets/bootstrap/_variables.scss) file.
+
+```yaml
+bootstrapCustomizations: ./path/to/bootstrap/customizations.scss
+```
+
+#### `appStyles`
+
+Default: `disabled`
+
+Import your custom styles here. Usually this endpoint-file contains list of `@imports` of your application styles.
+
+```yaml
+appStyles: ./path/to/your/app/styles/endpoint.scss
+```
+
+#### `styles`
+
+Default: all
+
+Bootstrap styles.
+
+```yaml
+styles:
+  mixins: true
+  normalize: true
+  ...
+
+# or enable/disable all of them:
+styles: true / false
+```
+
+#### `scripts`
+
+Default: all
+
+Bootstrap scripts.
+
+```yaml
+scripts:
+  transition: true
+  alert: true
+  ...
+
+# or enable/disable all of them:
+scripts: true / false
+```
+
+### Bootstrap 4
+
+There is only one additional option for Bootstrap 4:
+
+#### `useFlexbox`
+
+Default: `true`
+
+Enable / disable flexbox model.
+
+```yaml
+useFlexbox: true
+```
+
+## Additional configurations
+
+#### Paths to custom assets
+
+If you use `bootstrap-loader` to load your styles (via `preBootstrapCustomizations`, `bootstrapCustomizations` & `appStyles`) and you load custom assets (fonts, images etc.), then you can use relative paths inside `url` method (relative to SASS file, from which you load asset).
+
+This was made possible thanks to [resolve-url-loader](https://github.com/bholloway/resolve-url-loader). In common case you don't have to do anything special to apply it — we are doing it internally (just don't forget to install it). But if you want to use its custom settings, please provide it explicitly via `styleLoaders` option in `.bootstraprc`:
```

Comment:
We need an example of this.

Is the relative URL to the sass file? to the generated sass file during deployment? This is not clear.